Search for PAM policies in the following locations, in order:

	${pecan_sysconfdir}/pam.d
	${pecan_sysconfdir}/pam.conf

--- lib/openpam_configure.c.orig	2012-05-26 19:02:56.000000000 -0400
+++ lib/openpam_configure.c	2012-12-03 14:17:46.000000000 -0500
@@ -308,11 +308,13 @@
 	return (-1);
 }
 
+#ifndef SYSCONFDIR
+#define SYSCONFDIR	"/usr/local/etc"
+#endif
+
 static const char *openpam_policy_path[] = {
-	"/etc/pam.d/",
-	"/etc/pam.conf",
-	"/usr/local/etc/pam.d/",
-	"/usr/local/etc/pam.conf",
+	SYSCONFDIR "/pam.d/",
+	SYSCONFDIR "/pam.conf",
 	NULL
 };
 
